app/models/schemas/doi/funding_reference.json

```diff
@@ -0,0 +1,35 @@
+{
+  "title": "FundingReference",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "properties": {
+    "funderName": {
```

Comment:
It looks like funderName cannot be null or empty:

```
<xs:element name="funderName" minOccurs="1" maxOccurs="1">
<xs:annotation>
<xs:documentation>Name of the funding provider.</xs:documentation>
</xs:annotation>
<xs:simpleType>
<xs:restriction base="nonemptycontentStringType"/>
</xs:simpleType>
</xs:element>
```

Should this be `"type": ["string"]` ?